spec/index.html

```diff
@@ -628,13 +625,24 @@ <h3>IRIs</h3>
         characters, %-encoding of octets not allowed in URIs, and
         Punycode-encoding of domain names.</p>
 
-      <p><strong>Relative IRIs:</strong>
+      <p><strong>URLs:</strong>
+        The [[[URL]]] is largely compatible with [[RFC3987]] IRIs,
+        but is based on a processing model important for implementation
+        within Web browers and cannot be easily described using an [[ABNF]] grammar.
+        Additionally, URLs are processed,
+        for example, by replacing percent-encoded characters with their unencoded values.
+        Within RDF, IRIs are unprocessed.
```

Comment:
```suggestion
```
I don't think this is necessary - it's covered by "processing model" above. We only need a comment in RDF Concepts - not touching on an analysis. RFC3986 does talk about reversing percent-encoding when dereferencing a URI to access a resource.